static/js/services/enketo.js

```diff
@@ -24,47 +23,30 @@ angular.module('inboxServices').service('Enketo',
     var FORM_ATTACHMENT_NAME = 'xml';
     var REPORT_ATTACHMENT_NAME = this.REPORT_ATTACHMENT_NAME = 'content';
 
-    var replaceJavarosaMediaWithLoaders = function(formInternalId, form) {
```

Comment:
Yeah good idea. At the moment it's technically possible to have two forms with the same internal id...